test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -676,4 +705,30 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			By.css( '.edit-post-header .table-of-contents button' )
 		);
 	}
+
+	async dismissNotices() {
+		const locator = By.css( '.components-snackbar[aria-label="Dismiss this notice"]' );
+		const notices = await this.driver.findElements( locator );
+		await Promise.all( notices.map( ( notice ) => notice.click() ) );
+		await driverHelper.waitUntilElementNotLocated( this.driver, locator );
+	}
+
+	async insertBlockOrPatternViaBlockAppender( name, container = 'Group' ) {
+		const containerBlockId = await this.addBlock( container );
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( `#${ containerBlockId } .block-editor-button-block-appender` )
+		);
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( '.block-editor-inserter__quick-inserter .block-editor-inserter__search-input' ),
+			name
+		);
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css(
+				'.block-editor-inserter__quick-inserter .block-editor-block-types-list__item, .block-editor-inserter__quick-inserter .block-editor-block-patterns-list__item'
```

Comment:
Similar to the above, let's name these long selectors so we know what's being clicked.